pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/UsingPreparedStatementAndMetaDataObjects/Assignment5.java

```diff
@@ -0,0 +1,62 @@
+package com.subho.wipro.pjp.tm08.UsingPreparedStatementAndMetaDataObjects;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Assignment5 {
+	
+	public static void main(String[] args) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'main(String)' has an NPath complexity of 270](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)